src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePumpOptions.cs

```diff
@@ -46,15 +46,16 @@ public class AzureServiceBusMessagePumpOptions
         /// <remarks>
         ///     Requires 'Manage' permissions on the Azure Service Bus Topic to create and delete these subscriptions.
         /// </remarks>
-        public bool IncludeTopicSubscription { get; set; }
+        public TopicSubscription TopicSubscription { get; set; }
 
         /// <summary>
         ///     Default settings
         /// </summary>
         internal static AzureServiceBusMessagePumpOptions Default => new AzureServiceBusMessagePumpOptions
         {
             AutoComplete = true,
-            JobId = Guid.NewGuid().ToString()
+            JobId = Guid.NewGuid().ToString(),
+            TopicSubscription = TopicSubscription.None
```

Comment:
I would turn on subscription creation by default

Reply:
Creation but not deletion?

Reply:
Jup

Reply:
Yeah, is OK, I just alwasy think in a "clean after you're done"-approach 🗑

Reply:
Let's do that indeed, better